.github/workflows/ci.yml

```diff
@@ -0,0 +1,164 @@
+name: CI
+
+on: [pull_request, push]
+
+jobs:
+  build-docker:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - container: wpilib/roborio-cross-ubuntu:2020-18.04
+            artifact-name: Athena
+            build-options: "-Ponlylinuxathena"
+          - container: wpilib/raspbian-cross-ubuntu:10-18.04
+            artifact-name: Raspbian
+            build-options: "-Ponlylinuxraspbian"
+          - container: wpilib/aarch64-cross-ubuntu:bionic-18.04
+            artifact-name: Aarch64
+            build-options: "-Ponlylinuxaarch64bionic"
+          - container: wpilib/ubuntu-base:18.04
+            artifact-name: Linux
+            build-options: ""
+    name: "Build - ${{ matrix.artifact-name }}"
+    runs-on: ubuntu-latest
+    container: ${{ matrix.container }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Fetch all history and metadata
+        run: git fetch --prune --unshallow
+      - name: Build with Gradle
+        run: ./gradlew build -PbuildServer ${{ matrix.build-options }}
+      - uses: actions/upload-artifact@v2
+        with:
+          name: ${{ matrix.artifact-name }}
+          path: build/allOutputs
+
+  build-host:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: windows-latest
+            artifact-name: Win64
+            architecture: x64
+          - os: windows-latest
+            artifact-name: Win32
+            architecture: x86
+          - os: macos-latest
+            artifact-name: macOS
+            architecture: x64
+    name: "Build - ${{ matrix.artifact-name }}"
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Fetch all history and metadata
+        run: git fetch --prune --unshallow
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 11
+          architecture: ${{ matrix.architecture }}
+      - name: Build with Gradle
+        run: ./gradlew build -PbuildServer
+      - uses: actions/upload-artifact@v2
+        with:
+          name: ${{ matrix.artifact-name }}
+          path: build/allOutputs
+
+  build-cmake:
+    name: "Build - CMake"
+    runs-on: ubuntu-latest
+    container: wpilib/roborio-cross-ubuntu:2020-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: configure
+        run: mkdir build && cd build && cmake -DWITHOUT_ALLWPILIB=OFF ..
+      - name: build
+        run: cd build && make -j3
+
+  build-vcpkg:
+    name: "Build - vcpkg"
```

Comment:
The display name with Azure Pipelines was "Windows_CMake". Maybe this should be something named similar to clarify to PR authors that this is running a CMake build on Windows?